[CS2113T-F14-4] NUSchedule

Lee-Juntong opened the pull request:
NUSchedule helps students/staffs in NUS manage tasks of their daily routine, as well as help them to find the shortest route to their destination. It is optimized for CLI users so that frequent tasks can be done faster by typing in commands.

Chongjx commented on docs/DeveloperGuide.md:
```diff
@@ -1,26 +1,237 @@
+The `Model` component stores an ArrayList, events, that represents the current list of events.
+
+### 3.6 Location
+![Location class diagram](diagrams/Location.jpeg)
+*Figure 3.6.1 Class diagram for location component*
+
+__API:__ `LocationList.java` <br>
+
+The `LocationList` is made up of an ArrayList of Locations, which is a type of variable that stores different information 
+for different types of location (eg. `Building`, `Hostel`, etc.). However, do note that any place that is 
+not within the list of saved locations will be saved as type `OutOfNUS`, and would not be saved into the `location.txt` 
+file. The location will still be part of the location list before the app closes, and it will also be 
+saved as part of the Event information.
```
Is the Location class suppose to be an abstract class? Based on the enumeration, each location is of a certain type (building, hostel, LT or OutOfNUS) and there are classes for each of them, it looks like Location is an abstract class. Also, I think there should be an association between the Location and the enumeration

![image](https://user-images.githubusercontent.com/8433829/97528558-0496d400-19e9-11eb-9b9d-5579241bd38f.png)

Chongjx commented on docs/DeveloperGuide.md:
```diff
@@ -1,26 +1,237 @@
+and their information. Although the variable is declared in the main NUScheule class, it is only referenced 
+and not edited in any part of the program.
+
+<br>The `Location` component 
+* stores information about various types of locations 
+* prints the list of locations that is saved in the data file 
+* checks if a location is saved in the list and returns the location when asked
```
Does the Location component stores the information about the various types of locations? Sounds like it this is the LocationList component.

Chongjx commented on docs/DeveloperGuide.md:
```diff
@@ -1,26 +1,237 @@
+each other to run the program. 
+
+### 3.1 Architecture
+![architecture](../diagrams/architecture.png)
+
+The Architecture Diagram above provides a high-level view of the design of NUSchedule. The app can be broken down into 
+5 different components:  
+1. Main: Initializes the other components and connects them with each other.
+2. UI: Manages the User Interface that the user interacts with.
+3. Logic: Interprets user commands.
+4. Storage: Reads data from and writes data to the hard disk.
+5. Model: Stores the data the app uses in memory.
+
+__How the architecture components interact with each other__  
+The sequence diagram below shows how each individual component interacts with each other when the user inputs a command.
+![sequence](../diagrams/ArchitectureSequence.png)
```
Unable to load architecture and sequence images.

![image](https://user-images.githubusercontent.com/8433829/97529796-a4556180-19eb-11eb-9b7c-ec9d934f1687.png)

Chongjx commented on docs/DeveloperGuide.md:
```diff
@@ -1,26 +1,237 @@
+![](diagrams/Locate.jpg)
+*Figure 4.4.1 Sequence diagram for locate function*
+
+### 4.5 Reminders
+
+### 4.6 Sort events
+
+### 4.7 View events
+
+## 5. Documentation
+
```
Content are missing

Chongjx commented on docs/DeveloperGuide.md:
```diff
@@ -1,26 +1,237 @@
+![](diagrams/Locate.jpg)
+*Figure 4.4.1 Sequence diagram for locate function*
+
+### 4.5 Reminders
+
+### 4.6 Sort events
+
+### 4.7 View events
+
+## 5. Documentation
+
+### 5.1 Product scope
+__Target user profile:__
+* NUS student or staff
+* able to type quickly
+* has quite a number of events to keep track of  
+* prefers to use desktop apps
+* prefers using Command Line Interface (CLI) apps
+* prefers typing instead of mouse interactions
+
+### 5.2 Value proposition
 
 {Describe the value proposition: what problem does it solve?}
 
-## User Stories
+### 5.3 User Stories
 
 |Version| As a ... | I want to ... | So that I can ...|
 |--------|----------|---------------|------------------|
-|v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
-|v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
-
+|1.0|busy individual|keep track of both personal and school activities|avoid clashing events
+|1.0|freshman|know module details and lesson venues|be on time for my activities
+|1.0|forgetful person|be reminded of the deadlines for my assignments|submit on time
+|1.0|tutor|know the estimated time for my students to travel to their next class|pace my lesson suitably
+|1.0|hardworking student|track how much time I have spent studying\allocate my time efficiently
+|2.0|exchange student|know the optimal path to reach my next destination|avoid getting lost
+|2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
+|2.0|professor|Know whether my students have another lesson after mine and the expected time of travelling|Pace my lesson appropriately
```
Table format is broken

![image](https://user-images.githubusercontent.com/8433829/97530464-27c38280-19ed-11eb-8f96-f50888248c64.png)

Chongjx commented on docs/DeveloperGuide.md:
```diff
@@ -1,26 +1,237 @@
+ 2.	The command execution affects the EventList (e.g. clearing the list).
+ 3.	The result passes back to the UI, which then displays relevant feedback to the user.  
+
+### 3.5 Model
+__API__:`EventList.java`   
+The `Model` component stores an ArrayList, events, that represents the current list of events.
+
```
Lack of description and diagrams.

![image](https://user-images.githubusercontent.com/8433829/97530729-b59f6d80-19ed-11eb-820a-bc2166b0ef50.png)

zongxian-ctrl commented on docs/DeveloperGuide.md:
```diff
@@ -1,26 +1,237 @@
+
+### 3.6 Location
+![Location class diagram](diagrams/Location.jpeg)
+*Figure 3.6.1 Class diagram for location component*
```
Some formatting error might consider to review it before the final version update.

zongxian-ctrl commented on docs/DeveloperGuide.md:
```diff
@@ -1,26 +1,237 @@
+* Returns the location if valid
+* Prints an error message if invalid   
+
+The sequence diagram below shows exactly which methods, from which classes, are called to obtain the required location. 
+![](diagrams/Locate.jpg)
+*Figure 4.4.1 Sequence diagram for locate function*
```
Should the lifeline of EventList and LocationList be extended longer?

zongxian-ctrl commented on docs/DeveloperGuide.md:
```diff
@@ -1,26 +1,237 @@
+
+### 5.1 Product scope
+__Target user profile:__
+* NUS student or staff
+* able to type quickly
+* has quite a number of events to keep track of  
+* prefers to use desktop apps
+* prefers using Command Line Interface (CLI) apps
+* prefers typing instead of mouse interactions
```
Consider making it one sentence instead? For easier readability

chocomango reviewed (commented):
*Invalid review*
Sorry I submitted multiple reviews instead of comments, could the team kindly remove this?
Thank you.

chocomango reviewed (commented):
*Invalid review*
Sorry I submitted multiple reviews instead of comments, could the team kindly remove this?
Thank you.

chocomango commented on docs/DeveloperGuide.md:
```diff
@@ -1,26 +1,300 @@
+whether there is any event has overlap timing with the newly added event.
+
+The sequence diagram below shows the process of adding a new event.
+![AddCommand Sequence Diagram](diagrams/AddCommand.png)<br>
```
![image](https://user-images.githubusercontent.com/19763146/97657210-3faa0d80-1aa4-11eb-94de-2e7c6d3f425d.png)
Maybe for the objects in sequence diagram, you could use the instance:class format.
E.g. _ui:Ui_ ,_command:AddCommand_ and _:NusSchedule_

chocomango replied:
![image](https://user-images.githubusercontent.com/19763146/97656848-6b78c380-1aa3-11eb-9be6-dacb706957f0.png)
Should the invocation arrow in the sequence diagram be solid lines instead?

chocomango commented on docs/DeveloperGuide.md:
```diff
@@ -1,26 +1,300 @@
+each other to run the program. 
+
+### 3.1 Architecture
+![architecture](diagrams/architecture.png)<br>
```
![image](https://user-images.githubusercontent.com/19763146/97656527-a4fcff00-1aa2-11eb-9251-5b8cab5a0948.png)
After reading Ui.java, I believe the Ui component did not call parser or command. Perhaps, there should not be a directional arrow from Ui component to Logic component ?

chocomango commented on docs/DeveloperGuide.md:
```diff
@@ -1,26 +1,300 @@
+<br><br>
 
+## Table of Contents
+1. [Introduction](#1-introduction)
```
The hyperlinks are not working on GitHub pages as there will changes when GitHub convert markdown into HTML files. In this case, changing it to (#introduce) will work on pages but not markdown. Maybe the team could review this and find the best way to make it work?

Lee-Juntong replied:
Fixed. Inspired by https://raw.githubusercontent.com/AY2021S1-CS2113T-F11-1/tp/master/docs/DeveloperGuide.md

chocomango commented on docs/DeveloperGuide.md:
```diff
@@ -1,26 +1,300 @@
+entering `edit 1 class <new event description>` to edit the one existing event.
+
+Step 3. `editCommand()` function replaces the original event with the edited one. 
+![EditCommand Sequence Diagram](diagrams/EditCommand.png)<br>
```
![image](https://user-images.githubusercontent.com/19763146/97658034-4b96cf00-1aa6-11eb-9e9a-d195db491c86.png)
Since the components are already colour coded in the architecture diagram, perhaps the team could use the same colour for the background for the packages too?

chocomango reviewed (commented):
Just a few comments on diagrams.

xuche123 committed:
Merge pull request #147 from xuche123/EditCommand

Move new information parsing to edit command

xuche123 committed:
Fix bug in editing of an event into a personal event

xuche123 committed:
Merge pull request #148 from xuche123/EditCommand

Fix bug in editing of an event into a personal event

xuche123 committed:
Fix bug in conversion of assignment into other type

xuche123 committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-F14-4/tp into EditCommand

xuche123 committed:
Merge pull request #149 from xuche123/EditCommand

Edit command

Lee-Juntong committed:
Merge pull request #15 from AY2021S1-CS2113T-F14-4/master

Update

xuche123 committed:
Add location sorting

xuche123 committed:
Merge pull request #150 from xuche123/SortCommand

Add location sorting

Lee-Juntong committed:
study time is now available.
consider academics related personal event in next commit

Lee-Juntong committed:
study time now available.
consider academics related personal event in the next commit

Lee-Juntong committed:
Better reminder ( considers done/ not done)

Lee-Juntong committed:
Added selfStudy class

Lee-Juntong committed:
Merge pull request #154 from Lee-Juntong/StudyTime

Study time

xuche123 committed:
Allow edits for new class

xuche123 committed:
Merge pull request #155 from xuche123/EditCommand

Allow edits for new class

Lee-Juntong committed:
update Storage of self study. use equalsIgnorecase instead of equals to allow more valid input

Lee-Juntong committed:
Merge pull request #16 from AY2021S1-CS2113T-F14-4/master

update

Lee-Juntong committed:
Merge branch 'master' into StudyTime

# Conflicts:
#	src/main/java/event/SelfStudy.java
#	src/main/java/eventlist/EventList.java
#	src/main/java/parser/Parser.java

Lee-Juntong committed:
Merge pull request #156 from Lee-Juntong/StudyTime

update Storage of self study. use equalsIgnorecase instead of equals …

Lee-Juntong committed:
Merge pull request #17 from AY2021S1-CS2113T-F14-4/master

Update

jiaaaqi committed:
load location data without text file

jiaaaqi committed:
fixed locate command

jiaaaqi committed:
fixed coding style

jiaaaqi committed:
Merge pull request #158 from jiaaaqi/master

Fixed locate and storage

Lee-Juntong committed:
implement Repeat Command

Lee-Juntong committed:
Merge pull request #18 from AY2021S1-CS2113T-F14-4/master

Update

Lee-Juntong committed:
coding style

Lee-Juntong committed:
Merge branch 'master' into RepeatClasses

# Conflicts:
#	src/main/java/parser/Parser.java

Lee-Juntong committed:
Merge pull request #159 from Lee-Juntong/RepeatClasses

Repeat Command

Lee-Juntong committed:
Merge pull request #19 from AY2021S1-CS2113T-F14-4/master

Update

xuche123 committed:
Add more testing for EditCommand

xuche123 committed:
Fix checkstyle

xuche123 committed:
Add testing for sort command

xuche123 committed:
Fix checkstyle

xuche123 committed:
Merge pull request #160 from xuche123/EditCommand

Edit Command and Sort Command

xuche123 committed:
Add PPP

xuche123 committed:
Rename PPP

xuche123 committed:
Merge pull request #161 from xuche123/EditCommand

Add PPP

max-wunan committed:
-added AutoClearCommand class
-added autoclear command in parser

max-wunan committed:
-resolving conflicts

max-wunan committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-F14-4/tp.git into autoclear

# Conflicts:
#	src/main/java/parser/Parser.java

max-wunan committed:
-added clearbefore function in eventlist
(the autoclear function is called everytime the user type in autoclear

max-wunan committed:
-added Clearbefore command
-added EmptyClearDateException
-added NoDateBeforeException
-added NoEventOneMonthAgoException
-updated parser for autoclear & clearbefore command

max-wunan committed:
Merge pull request #162 from max-wunan/autoclear

autoclear and clearbefore function

max-wunan committed:
-updated the clearbefore successul message

max-wunan committed:
updated printHelp() function in ui

max-wunan committed:
Merge pull request #163 from max-wunan/helpupdate

to update help message to the current version

max-wunan committed:
format & bug fixes

xuche123 committed:
Update userguide

max-wunan committed:
-more format editing

xuche123 committed:
Minor edits

max-wunan committed:
Merge pull request #164 from max-wunan/bugfixes

format & bug fixes

xuche123 committed:
Merge branch 'master' into EditCommand

xuche123 committed:
Merge pull request #167 from xuche123/EditCommand

Update userguide

xuche123 committed:
Fix spacing

xuche123 committed:
Merge pull request #168 from xuche123/EditCommand

Fix spacing

max-wunan committed:
-adding ppp for wu nan

max-wunan committed:
ppp for wu nan

max-wunan committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-F14-4/tp.git into bugfixes

Lee-Juntong committed:
Merge pull request #20 from AY2021S1-CS2113T-F14-4/master

Update

max-wunan committed:
-updated userguide about command autoClear & clearBefore

max-wunan committed:
Merge pull request #169 from max-wunan/documentation

Documentation

Lee-Juntong committed:
Edit PPP to the draft version

Lee-Juntong committed:
update the photo

xuche123 committed:
Update xuche123.md

xuche123 committed:
Update xuche123.md

xuche123 committed:
Update AboutUs.md

xuche123 committed:
Update AboutUs.md

xuche123 committed:
Update xuche123.md

xuche123 committed:
Merge pull request #170 from Lee-Juntong/master

Update PPP and AboutUs

xuche123 committed:
Fix diagrams

jiaaaqi committed:
add location tests

xuche123 committed:
Update DG

xuche123 committed:
Merge pull request #171 from xuche123/final

Update developer guide

xuche123 committed:
Update DeveloperGuide.md

xuche123 committed:
Update DeveloperGuide.md

xuche123 committed:
Update xuche123.md

xuche123 committed:
Update xuche123.md

xuche123 committed:
Update xuche123.md

xuche123 committed:
Update xuche123.md

xuche123 committed:
Update xuche123.md

jiaaaqi committed:
add ppp

Lee-Juntong committed:
Includes some locations in UI for printing

Lee-Juntong committed:
Merge pull request #172 from jiaaaqi/master

add location testing and ppp

Lee-Juntong committed:
Merge pull request #21 from AY2021S1-CS2113T-F14-4/master

Update

Lee-Juntong committed:
Better Coding Style

Lee-Juntong committed:
Merge pull request #173 from Lee-Juntong/master

Update UI to print some location info

Lee-Juntong committed:
Added the Command to print the locations available at the selected area

max-wunan committed:
resolving conflicts

max-wunan committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-F14-4/tp.git into documentation

Lee-Juntong committed:
Complete the print area locations method

Lee-Juntong committed:
Updated PPP

max-wunan committed:
-update dg/ug for clear function
-update clear function sequence diagram
-update help function sequence diagram

jiaaaqi committed:
added code fragments into user guide

Lee-Juntong committed:
Merge pull request #174 from Lee-Juntong/PrintAreaLocations

Print area locations

max-wunan committed:
Merge pull request #175 from max-wunan/documentation

Documentation

jiaaaqi committed:
fix style error

jiaaaqi committed:
Merge pull request #176 from jiaaaqi/master

added code fragments into user guide

max-wunan committed:
-draft ppp for wu nan

max-wunan committed:
Merge pull request #177 from max-wunan/documentation

PPP for wunan

max-wunan committed:
Update AboutUs.md

to make ppp page available from AboutUs

max-wunan committed:
Update AboutUs.md

max-wunan committed:
Update AboutUs.md

to make all existing ppp available on AboutUs

jiaaaqi committed:
added more location data

jiaaaqi committed:
fixed minor error in locate command

jiaaaqi committed:
updated ppp

max-wunan committed:
-update aboutus to make ppp link available
-added page breaker to create cover page for ug/dg

max-wunan committed:
update cover page

jiaaaqi committed:
fixed spacing error

max-wunan committed:
Merge pull request #179 from max-wunan/documentation

Documentation

jiaaaqi committed:
Merge pull request #178 from jiaaaqi/master

added more location data and update ppp

max-wunan committed:
updated wu nan's ppp

jiaaaqi committed:
updated about us

jiaaaqi committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-F14-4/tp

# Conflicts:
#	docs/AboutUs.md

max-wunan committed:
updated PR reviewed for wu nan's ppp

jiaaaqi committed:
Merge pull request #181 from jiaaaqi/master

update about us

max-wunan committed:
Merge pull request #180 from max-wunan/documentation

updated wu nan's ppp

jiaaaqi committed:
Update jiaaaqi.md

max-wunan committed:
added bug reporting for other teams in ppp

Lee-Juntong committed:
Enhanced both auto clear (by making it an option) and clear before (considers end date instead of start date)

Lee-Juntong committed:
coding style

jiaaaqi committed:
fixed reposense link

Lee-Juntong committed:
coding style

Lee-Juntong committed:
Merge pull request #182 from Lee-Juntong/Enhance_Auto_Clear

Enhance auto clear

Lee-Juntong committed:
Merge pull request #22 from AY2021S1-CS2113T-F14-4/master

Update

madbeez committed:
Updated DG/UG/PPP

Lee-Juntong committed:
update PPP

Lee-Juntong committed:
Merge pull request #184 from Lee-Juntong/PPP

PPP

madbeez committed:
Merge branch 'master' into master

madbeez committed:
Merge pull request #183 from madbeez/master

Updated DG/UG/PPP

jiaaaqi committed:
fix repo sense link

madbeez committed:
Fixed typo in aboutus.md

jiaaaqi committed:
Merge pull request #185 from jiaaaqi/master

fix repo sense link

madbeez committed:
Update madbeez.md

Lee-Juntong committed:
Reject the user from marking an event with later end time compared to now as done. Of course, Assignments are not considered inside.

Lee-Juntong committed:
make the function doneEvent() work as expected

Lee-Juntong committed:
Merge pull request #186 from Lee-Juntong/Done_Enhancement

Done enhancement

Lee-Juntong committed:
Merge pull request #23 from AY2021S1-CS2113T-F14-4/master

Update

max-wunan committed:
update q&a

max-wunan committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-F14-4/tp.git into documentation

xuche123 committed:
Update xuche123.md

Lee-Juntong committed:
Added NullpointerException for printGreetingMessage()

jiaaaqi committed:
minor edits

jiaaaqi committed:
Merge pull request #187 from jiaaaqi/master

minor edits

xuche123 committed:
Update UserGuide.md

madbeez committed:
Fixed UG/PPP formatting

madbeez committed:
Updated PPP

madbeez committed:
Merge pull request #188 from madbeez/master

Updated UG/PPP

xuche123 committed:
Update xuche123.md

max-wunan committed:
-update q&a of ug

max-wunan committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-F14-4/tp.git into documentation

Lee-Juntong committed:
fixed the problem when UserInfo.txt is empty

max-wunan committed:
update ppp for wu nan

max-wunan committed:
Merge pull request #189 from max-wunan/documentation

Documentation

Lee-Juntong committed:
coding style

max-wunan committed:
update readme.md

Lee-Juntong committed:
Merge pull request #190 from Lee-Juntong/master

Fix a bug

max-wunan committed:
Merge pull request #191 from max-wunan/documentation

update readme.md

Lee-Juntong committed:
Merge pull request #24 from AY2021S1-CS2113T-F14-4/master

Update

max-wunan committed:
added logo/coverpage for ReadMe.md

max-wunan committed:
Merge pull request #192 from max-wunan/documentation

added logo/coverpage for ReadMe.md

max-wunan committed:
adding line breakers for ppp to make it more clean to view

max-wunan committed:
Merge pull request #193 from max-wunan/documentation

adding line breakers for ppp to make it more clean to view

max-wunan committed:
fix format error of ppp

max-wunan committed:
fix format

max-wunan committed:
Merge pull request #194 from max-wunan/documentation

Documentation

max-wunan committed:
remove the wrong diagram for autoclear

max-wunan committed:
Merge pull request #195 from max-wunan/documentation

remove the wrong diagram for autoclear

Lee-Juntong committed:
Update UG for repeat

Lee-Juntong committed:
Merge pull request #196 from Lee-Juntong/UGDG

UGDG as well as the problem with selfStudy

max-wunan committed:
format fix

max-wunan committed:
Merge pull request #197 from max-wunan/documentation

format fix

max-wunan committed:
update ug/dg for autoclear

max-wunan committed:
Merge pull request #198 from max-wunan/documentation

update ug/dg for autoclear

Lee-Juntong committed:
update UGDG

Lee-Juntong committed:
Merge pull request #25 from AY2021S1-CS2113T-F14-4/master

update

Lee-Juntong committed:
Merge branch 'master' into UGDG

# Conflicts:
#	docs/UserGuide.md

Lee-Juntong committed:
Merge pull request #199 from Lee-Juntong/UGDG

Ugdg

Lee-Juntong committed:
update DG

Lee-Juntong committed:
Merge pull request #200 from Lee-Juntong/UGDG

update DG

max-wunan committed:
added line breakers before diagrams

max-wunan committed:
Merge pull request #201 from max-wunan/documentation

added line breakers before diagrams

jiaaaqi committed:
dg content page

jiaaaqi committed:
Merge pull request #202 from jiaaaqi/master

dg content page

max-wunan committed:
add printAreaLocationsCommand png

Lee-Juntong committed:
Update DG

max-wunan committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-F14-4/tp.git into documentation

# Conflicts:
#	docs/DeveloperGuide.md

xuche123 committed:
Update DG

xuche123 committed:
Merge pull request #204 from xuche123/final

Update DG

Lee-Juntong committed:
Merge branch 'master' into UGDG

Lee-Juntong committed:
Merge branch 'master' into UGDG

Lee-Juntong committed:
Merge pull request #203 from Lee-Juntong/UGDG

Update DG

max-wunan committed:
update dg for print areas

max-wunan committed:
format fix

max-wunan committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-F14-4/tp.git into documentation

# Conflicts:
#	docs/DeveloperGuide.md

jiaaaqi committed:
update done function

max-wunan committed:
Merge pull request #205 from max-wunan/documentation

Documentation

jiaaaqi committed:
Merge pull request #206 from jiaaaqi/master

update done function

madbeez committed:
Updated DG

madbeez committed:
Updated DG

Lee-Juntong committed:
Merge pull request #29 from AY2021S1-CS2113T-F14-4/master

update

madbeez committed:
Merge pull request #207 from madbeez/master

Updated DG

jiaaaqi committed:
dg (find and delete)

Lee-Juntong committed:
DG

jiaaaqi committed:
Merge pull request #208 from jiaaaqi/master

dg (find and delete)

Lee-Juntong committed:
Merge pull request #209 from Lee-Juntong/master

dg

Lee-Juntong committed:
Merge pull request #30 from AY2021S1-CS2113T-F14-4/master

dg

max-wunan committed:
update dg

max-wunan committed:
Merge pull request #210 from max-wunan/documentation

update dg

jiaaaqi committed:
dg links

jiaaaqi committed:
Merge pull request #211 from jiaaaqi/master

dg links

Lee-Juntong committed:
Update DeveloperGuide.md

Lee-Juntong committed:
Rename Lee-Juntong.md to lee-juntong.md

jiaaaqi committed:
links

Lee-Juntong committed:
Update DeveloperGuide.md

Lee-Juntong committed:
Update DeveloperGuide.md

jiaaaqi committed:
next page

jiaaaqi committed:
update dg

Lee-Juntong committed:
Merge pull request #31 from AY2021S1-CS2113T-F14-4/master

dg

Lee-Juntong committed:
some

max-wunan committed:
remove numbering of DG

max-wunan committed:
Merge pull request #212 from max-wunan/documentation

remove numbering of DG

max-wunan committed:
Update DeveloperGuide.md

jiaaaqi committed:
links

Lee-Juntong committed:
Merge branch 'master' of https://github.com/Lee-Juntong/tp

# Conflicts:
#	docs/DeveloperGuide.md

Lee-Juntong committed:
Update DeveloperGuide.md

max-wunan committed:
alignment

max-wunan committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-F14-4/tp.git into documentation

Lee-Juntong committed:
Update lee-juntong.md

Lee-Juntong committed:
Update lee-juntong.md

max-wunan committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-F14-4/tp.git into documentation

xuche123 committed:
Update UserGuide.md

Lee-Juntong committed:
clear DG

Lee-Juntong committed:
Merge branch 'master' into master

Lee-Juntong committed:
Merge pull request #32 from AY2021S1-CS2113T-F14-4/master

Update

madbeez committed:
Update UserGuide.md

Lee-Juntong committed:
UG author

Lee-Juntong committed:
Merge pull request #213 from Lee-Juntong/master

UG

madbeez committed:
Update UserGuide.md

jiaaaqi committed:
add name to sections

jiaaaqi committed:
accidentally deleted a line

jiaaaqi committed:
Merge pull request #214 from jiaaaqi/master

add name to section

max-wunan committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-F14-4/tp.git into documentation

max-wunan committed:
adding author

max-wunan committed:
Merge pull request #215 from max-wunan/documentation

Documentation

xuche123 committed:
Update UserGuide.md

Lee-Juntong committed:
Update README.md

Lee-Juntong committed:
Update README.md

Lee-Juntong committed:
Update README.md
